tests/docker-compose-int-test.yml

```
@@ -44,7 +44,7 @@ services:
   # Will run `runRobotTests`
   #
   ehrbase-integration-tests:
-    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest}
+    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest-feature-CDR-1466_robot_tests_dv_multimedia}
```

Changing back to latest
```suggestion
    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest}
```